src/sqlfluff/rules/L047.py

```diff
@@ -87,7 +93,64 @@ def _eval(self, segment, **kwargs):
                         ),
                     ],
                 )
-            if not self.prefer_count_1 and f_content[0].is_type("expression"):
+            elif self.prefer_count_1 and f_content[0].is_type("expression"):
+                expression_content = [
+                    seg for seg in f_content[0].segments if not seg.is_meta
+                ]
+                if (
+                    len(expression_content) == 1
+                    and expression_content[0].is_type("literal")
+                    and expression_content[0].raw == "0"
+                ):
+                    return LintResult(
+                        anchor=segment,
```

Comment:
I notice it says this line is not covered by tests?

Reply:
Just submitted some more tests in https://github.com/sqlfluff/sqlfluff/pull/1263/commits/7ed3c89982b08b990499425ce15245724fb80559. will monitor to make sure coverage is back to normal.

Reply:
Looks better now.